app/src/main/res/layout/add_transaction_fragment.xml

```diff
@@ -27,21 +27,21 @@
             android:layout_height="wrap_content"
             android:ems="10"
             android:inputType="textPersonName"
-            android:text="@string/AmountTxt" />
+            android:text="@string/text_amount" />
```

Comment:
Changed to new name

Reply:
I'd go with lowerCamelCase, which is common.  So... textAmount.  Or just... amount